fml/synchronization/sync_switch.cc

```diff
@@ -0,0 +1,43 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/fml/synchronization/sync_switch.h"
+
+namespace fml {
+
+SyncSwitch::Handlers::Handlers() : true_handler({}), false_handler({}) {}
```

Comment:
Do you see any benefit to that?  The more logic we put inside the .cc files the faster compilation will be (less parsing and recompiling when changing things).

Reply:
If more ivars or constructors are added, we have to go check that all constructors consistently default all their uninitialized ivars. We have messed up on this in the past (for a while the first frame rendered by the engine was of an undefined size because a new constructor did not initialize the default frame size to zero and we checked if we wanted to render a frame on a non-zero frame size).

> faster compilation

I'd rather optimize for maintainability (mentioned above) rather than compiler performance and just use Goma.